apps/emqx_dashboard_rbac/BSL.txt

```diff
@@ -0,0 +1,94 @@
+Business Source License 1.1
```

Comment:
Should it be placed in the lib-ee directory if it is a BSL application? Also, we should not pack a BSL application when doing releases.